### ISSUE TITLE
Add support for Zebra LinkOS PDF functionality

### ISSUE THREAD

**Issue by michaelrsweet** [michaelrsweet/lprint#110]
Investigate adding support for Zebra's PDF support in recent label printer products.

**Comment by milkman0007** [comment]
If link pdf is added would that also translate for other label printers too

**Comment by michaelrsweet** [comment]
@milkman0007 No, this would only add PDF support to printers that already provide it natively.
